docs/install/win/docker.md

```diff
@@ -33,6 +33,7 @@ docker pull gcr.io/prysmaticlabs/prysm/beacon-chain:latest
 
 This process will also install any related dependencies.
 
+> For advanced users, the beacon-chain and validator images with debugging tools bundled in can be fetched instead by appending `-alpine` to the end of the images in the `pull` commands above. For example: `docker pull .../prysm/validator:latest-alpine`.
```

Comment:
I think that alpine is probably recommended for every OS. Not sure why this is different here.

`-debug` is prysm built with debug symbols and without compiler optimizations. This process will be slower and perform worse.
`-alpine` is the same binaries, but with a different base image. Alpine base image comes with many tools pre-installed.  This image is only 5mb larger than the default one. https://hub.docker.com/_/alpine